### ISSUE TITLE
Maybe it's time for a new release?

### ISSUE THREAD

**Issue by mrquantumoff** [mackwic/colored#114]

**Comment by hwittenborn** [comment]
I'd like to start using `CustomColor` in my package, I'm definitely down with a new release.

**Comment by dhruvkb** [comment]
I'm slightly concerned that the library is no longer actively maintained and may be abandonware. The last release was way back in 2019 and the last commit was a little over a year ago in June 2022.

**Comment by hwittenborn** [comment]
Hmm it looks like the maintainer's GitHub is still pretty active, have you just been busy with other stuff @mackwic? I'm sure some of us (at least me anyway) would be more than glad to help with things like this if you'd need.

**Comment by mackwic** [comment]
Hi everyone. Thanks a lot for the notification @hwittenborn.

To clarify the situation:
@dhruvkb : **I consider Colored to be stablized in maintenance mode**, most of the features are there. It's perfectible, there are always people asking for more, but I feel like it's in a good enough state. Rust being a very stable language, and Colored having few dependencies means that the maintenance burden is low and there is not a great need of new releases. Do not fear the old versions. ;)

That said, I agree that Colored would profit a lot from some maintenance care. Styles update, release the new features, and improve the DX thanks to the better and powerful type system we have now rather than the one we had in 2015.

This is care I cannot provide as I'm funding a new company, so I'm glad to accept all the help. @hwittenborn I'd like to add you as a committer, let's organize ! Can I send you an email to discuss the details with you?

**Comment by hwittenborn** [comment]
Yeah, that'd be great @mackwic! You can reach me at [hunter@hunterwittenborn.com](mailto:hunter@hunterwittenborn.com).

**Comment by hwittenborn** [comment]
Hey @mackwic, did you ever email me? Just wanting to make sure it didn't bounce into spam or something.

**Comment by mackwic** [comment]
Sorry it's me, I've just sent you an email.

**Comment by hwittenborn** [comment]
Thanks @mackwic! Just sent an email back.

**Comment by hwittenborn** [comment]
This should be good to go now, I've gone ahead an made a release on crates.io. Feel free to send a message if y'all got anything else :)
